w12_manybody/topoorder.ipynb

```diff
@@ -68,9 +57,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We have worked hard to create topological models so far. The FQH system, which is the most topological in some sense, was also more obscure in terms of microscopics. Here we will follow Alexei Kitaev and right down a simple Hamiltonian that is obviously topological, but also relatively easy to analyse.\n",
+    "We have worked hard to create topological models so far. The FQH system, which is the most topological in some sense, was also more obscure in terms of microscopics. Here, we follow Alexei Kitaev and write down a simple Hamiltonian that is obviously topological, but also relatively easy to analyse.\n",
     "\n",
-    "The Hamiltonian we will write is that of a system of localized spin$-1/2$ electrons that live on the bonds of a square lattice. The particular Hamiltonian that Kitaev wrote down is:\n",
+    "The Hamiltonian consists of a system of localized spin$-1/2$ electrons that live on the bonds of a square lattice. The particular Hamiltonian that Kitaev wrote down is:\n",
```

Comment:
Sorry, I was slow editing my previous comment. The Hamiltonian doesn't consist of a system of spins. Should we rather write something like "Let's take a system of..."? It is somewhat colloquial, and a more formal version would be "Consider a system of...", but I think it's still OK.